src/MonthView/Weekdays.jsx

```diff
@@ -34,12 +35,13 @@ export default function Weekdays(props) {
       year, monthIndex, weekday - getDayOfWeek(beginOfMonth, calendarType),
     );
 
+    const weekdayClassName = `${className}__weekday`;
```

Comment:
Move to line 17; there's no need to re-create this variable on every render, let alone in a loop!